py-rse/git-cmdline.Rmd

```diff
@@ -4,15 +4,29 @@
 source(here::here("_common.R"))
 ```
 
-The first two volumes in this series introduced version control with Git
-using a graphical interface like [GitKraken][gitkraken] or [the RStudio IDE][rstudio-ide].
-These interfaces are actually wrappers around Git's original command-line interface,
-which allows us to access all of Git's features.
-This lesson describes how the basic cycle of add, commit, push, and pull works at the command line.
-
-To illustrate with the Zipf's Law project,
-we will start by tracking our work up to this point in Git.
-The project directory should currently include:
+A [version control system][version-control-system] records changes to files
+and helps people share their work with each other.
+These things can be done by emailing files to colleagues
+or by using "track changes" in Microsoft Word and Google Docs,
+but version control does both more accurately and efficiently.
+Originally developed to support software development,
+over the past fifteen years it has become the cornerstone
+of [reproducible research][reproducible-research].
+
+The most widely used version control system today is [Git][git].
+Many people first encounter it through a GUI like [GitKraken][gitkraken]
+or [the RStudio IDE][rstudio-ide],
+and we assume that you have used one of these to save your work before.
```

Comment:
In #389 we discussed not assuming that people have used a git GUI. To address this, #390 was merged a few days ago. You'll need to pull those changes into this branch.

#390 basically did two things - it added a general introduction about what version control is at the beginning, and then added content midway through the lesson about the staging area. With those additions, this chapter basically covers the same content as the Software Carpentry git lesson, which doesn't require that people have used a git GUI.

When you pull the changes from #390, there will be conflicts with the new introduction that you've written in this PR. I like your introduction better, so feel free to simply take the bits of my introduction that you like or delete my intro altogether.

Reply:
Oops - I had pulled those changes into my desktop repo but must have created the `git-cmdline` branch beforehand. I've merged them into this PR - please have a look.

Reply:
👍